content/1.7/back-office/dashboard/dashboard-configure.md

```diff
@@ -0,0 +1,57 @@
+---
+title: Dashboard Configure
+weight: 1
+---
+
+# Dashboard configure
+
+3 blocks are displayed on this page:
+
+ - **Average bank fees per payment method**
+ - **Average shipping fees per shipping method**
+ - **Other settings**
+
+## Demo mode
+
+ - **Demo mode**: This toggle allows to enable or disable fake data. By default, it is disabled. When enabled, fake data are used. When disabled, the real store's data are used. The store's data are gathered by PrestaShop's statistics system.
+The following tooltip is displayed: "This mode displays sample data so you can try your dashboard without real numbers."
+
+## Average bank fees per payment method
+
+4 Input fields are presented in this block, these are the default fields, if more payment methods are added to the shop, they will be displayed additionally here:
+
+ - **Bank transfer** - a text below the field shows: "Choose a fixed fee for each order placed in USD% with Bank transfer." Only numbers are allowed in this field, on the right of this field the currecncy that is set on the shop is displayed. If the field input is invalid, when trying to save will show the following: "The Bank transfer field is invalid."
+ - **Bank transfer** - a text below the field shows: "Choose a variable fee for each order placed in USD% with Bank transfer. It will be applied on the total paid with taxes." Only numbers are allowed in this field, on the right of the field there is a percentage sign. If the field input is invalid, when trying to save will show the following: "The Bank transfer field is invalid.".
```

Comment:
```suggestion
 - **Bank transfer** - a text below the field shows: "Choose a variable fee for each order placed in USD% with Bank transfer. It will be applied on the total paid with taxes." Only numbers are allowed in this field. On the right of this field, a percentage sign is displayed. If the field is invalid and when trying to save, the following error message is displayed: "The Bank transfer field is invalid.".
```